File: Data Structures & Algorithms/remove-element/submission-1.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        curr_count = 0
        eq_val = 0
        for i, num in enumerate(nums):
            if i >= len(nums)-eq_val:
                break
            if num == val:
                swap = True
                while swap and i != len(nums)-eq_val:
                    eq_val += 1
                    # swap curr and last(ish)
                    temp = nums[len(nums)-eq_val]
                    nums[len(nums)-eq_val] = num
                    nums[i] = temp
                    if nums[i] != val:
                        swap = False
                    # self.print_arr(nums)
        return len(nums)-eq_val

    def print_arr(self, nums: List[int]):
        for num in nums:
            print(nums)

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        int max_count = 0;
        int curr_count = 0;
        for (int i = 0; i < nums.Length; ++i) {
            if (nums[i] == 0) {
                max_count = Math.Max(max_count, curr_count);
                curr_count = 0;
            }
            else ++curr_count;
        }
        max_count = Math.Max(max_count, curr_count);
        return max_count;
    }
}